src/app-plaid/app-plaid.js

```diff
@@ -0,0 +1,1033 @@
+import express from 'express';
+import https from 'https';
+import { SecretName, secretsService } from '../services/secrets-service.js';
+import { handleError } from '../app-gocardless/util/handle-error.js';
+import { requestLoggerMiddleware } from '../util/middlewares.js';
+import validateSession from '../util/validate-user.js';
+import { Configuration, PlaidApi, Products, PlaidEnvironments } from 'plaid';
+import Airtable from 'airtable';
+
+import {
+  getUserInfo,
+} from '../account-db.js';
+
+const app = express();
+export { app as handlers };
+app.use(express.json());
+app.use(requestLoggerMiddleware);
+
+let REACT_APP_PLAID_CLIENT_ID = process.env.REACT_APP_PLAID_CLIENT_ID;
+let REACT_APP_PLAID_SECRET = process.env.REACT_APP_PLAID_SECRET;
+let REACT_APP_PLAID_REDIRECT_URL = process.env.REACT_APP_PLAID_REDIRECT_URL;
+
+const configuration = new Configuration({
+  basePath: PlaidEnvironments.sandbox, // or development/production
```

Comment:
We need to add a switch between Plaid's sandbox and production envs before launch